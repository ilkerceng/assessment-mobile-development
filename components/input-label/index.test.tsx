import React from "react";
import renderer from "react-test-renderer";
import InputLabel from "./index";

describe("<input-label />", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<InputLabel id={1} label={"test"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("has 1 child", () => {
    const tree = renderer.create(<InputLabel />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});
