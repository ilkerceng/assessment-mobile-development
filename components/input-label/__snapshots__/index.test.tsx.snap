// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<input-label /> renders correctly 1`] = `
<View>
  <Text>
     
    test
     
  </Text>
</View>
`;
